eng/common/pipelines/templates/steps/set-default-branch.yml

```diff
@@ -0,0 +1,7 @@
+steps:
```

Comment:
We should add the repo/remote name as a parameter and default it to origin. We should also add a parameter for the working directory so that people can make sure it is ran in the correct context. For example with those parameters I could have used this template in https://github.com/Azure/azure-sdk/commit/df13ed73e6164ba116a6a98518d12a3b74fa1eff.

Reply:
Working on updating the comment here and above.